### DISCUSSION TITLE
week 3

### DISCUSSION THREAD

**Discussion by DavidDHBW** [DavidDHBW/codespaces-jupyter#5] [General]
added an start function and you can now exit the game properly :)

additionaly the solution word is picked out of a list with strings

[code](https://github.com/DavidDHBW/codespaces-jupyter/blob/main/Python/Hangman/hangman.py)

**Comment by LeoEnglert** [comment]
Hey there,
if I try to access your provided Link, I get a 404.
Maybe you could fix that, so I can review your code 👁️👄👁️

**Reply by DavidDHBW** [reply]
oh okay

**Reply by LeoEnglert** [reply]
Ah thanks :)

Your code is well made and structured, I like it!
One thing you could do even better, would be if you stick with one language when naming your methods and variables.
For example `def NutzerEingabe():` is german but the rest is english.
But as I said, I really like your Game and the code you have written!

**Reply by DavidDHBW** [reply]
thanks for the feedback. My naming is indeed very messy

**Comment by LxisW** [comment]
Basic but cool game. Would put the code in different functions and not in the file somewhere for a more clear code.

**Reply by DavidDHBW** [reply]
thanks for the feedback. Your right I should consider structurize my Code better :)

**Comment by emmanuelyoussefyoussef** [comment]
Just played your game and lost... nevertheless a very creative game and a nice code 👌

**Reply by DavidDHBW** [reply]
thanks
